component/Card/style.css

```diff
@@ -0,0 +1,196 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+body {
+  font-family:san-serif;
+  background-color: rgb(151, 151, 151);
+  font-family: sans-serif;
+  display:flex;
+  gap: 1rem;
+  margin: 1rem;
+  flex-wrap: wrap;
+}
+
+/* text only card */
+
+
+.card-background {
+  width:15.625rem;
+  background-color:white;
+  border-radius: 0.313rem;
+}
+
+.card-title{
+  margin-bottom:0.5rem;
+}
+.card-description{
+  margin-top:0.5rem;
+}
+.small-text{
+  color:grey;
+}
+
+
+/* card with dismiss */
+
+.card-background{
+  position: relative;
+}
+
+.badge-icon{
+  height:2.5rem;
+  width:2.5rem;
+  position: absolute;
+  top: 10px;
+  right: 10px;
+  cursor: pointer;
+  padding: 0.5rem;
+  border-radius:50%;
+}
+
+.badge-icon:hover{
+  background-color: rgb(226, 224, 224);
```

Comment:
use consistent units for colors(may be you can convert grey and white to rgb)
Also, you can declare common colors(like white, grey, primary color etc.) as variables in some common file and reuse it in other components

Reply:
done